### DISCUSSION TITLE
What is DHU?

### DISCUSSION THREAD

**Discussion by tarasov503** [AndreyPavlenko/Fermata#438] [Q&A]
Sorry for the dumb question, but I've been using Fermata on my head unit through head unit reloaded app, but I keep seeing connect to DHU on these forums and am clueless

**Comment by AndreyPavlenko** [comment]
https://developer.android.com/training/cars/testing/dhu
If you use the head unit reloaded app, then you probably don't need DHU.

**Reply by tarasov503** [reply]
Thank you for the quick response. The only issue I seen to have, when connecting to my head unit through head unit reloaded, when I click on the Fermata Mirror app, most the time when it asks me to accept the screen mirror permission, the screen mirror permission will keep popping up several times, and more times than not, even after hitting accept several times, my head unit says mirror permissions rejected.. even though I've clicked accept several times. But then, after the second, sometimes third try, it happens to connect and work. Not sure why it's doing this. Didn't seem to have that issue when I was using my S23 FE. Now that I'm using a pixel 8 pro, it usually takes several attempts to connect

**Reply by AndreyPavlenko** [reply]
As mentioned [here](https://github.com/AndreyPavlenko/Fermata/discussions/432) you may disable this permission request with adb:
```
adb shell appops set me.aap.fermata.auto.dear.google.why PROJECT_MEDIA allow
```

**Comment by Planetiptv** [comment]
@AndreyPavlenko can u help me? 
Uconnect 7"
Fermata auto full screen but fermat mirror fs not full screen 
![IMG_20241222_130455_060835](https://github.com/user-attachments/assets/36c975ae-27a2-4e80-aa98-e420512f9787)
![IMG_20241222_130436_060801](https://github.com/user-attachments/assets/511fff46-9b35-4e1e-9556-bd3e1e56fa96)
